proxy/tests/telemetry.rs

```diff
@@ -519,19 +514,17 @@ mod outbound_dst_labels {
     fn controller_updates_addr_labels() {
```

Comment:
I don't think that's the case.

This test does the following:
1. publishes a dst
2. routes a request
3. publishes an updated dst
4. routes a request

at (2), we *must* get a dst to complete the routing.  at (4), however, i don't think that we can be guaranteed that the router has observed the update, since it can just route on the old data.  Typically, it will; but no guarantees about cross-thread visibility/ordering.

Reply:
Ah, I think you're right, nevermind. Oh well.